packages/webdriverio/src/scripts/resq.js

```diff
@@ -0,0 +1,38 @@
+export const waitToLoadReact = function waitToLoadReact () {
+    window.resq.waitToLoadReact()
+}
+
+export const react$ = function react$ (selector, props) {
+    const elems = window.resq.resq$$(selector)
+
+    if (Object.keys(props).length === 0) {
+        return elems[0]
+    }
+
+    let elem = [...elems].find((elem) => {
+        for (const [name, value] of Object.entries(props)) {
+            // logs.push(`check ${name} and ${value} for ${elem[name]}`)
+            if (elem.props[name] === value) {
```

Comment:
If you use `resq$(selector).byProps(props)` you will get the first element in which the props match. Accordingly, you could use `resq$$` to select all elements that match selector and props. 

Keep in mind, that as of right now, `byProps` and `byState` are deep matchers.

Reply:
That's awesome, I didn't know about this.